base_layer/core/src/chain_storage/lmdb_db/lmdb_db.rs

```diff
@@ -332,15 +332,40 @@ impl LMDBDatabase {
                     block_accum_data.deleted = DeletedBitmap { deleted };
                     self.update_block_accumulated_data(&write_txn, height, &block_accum_data)?;
                 },
+                UpdateDeletedBlockAccumulatedDataWithDiff {
```

Comment:
The `UpdateDeletedBlockAccumulateData` enum variant is most likely not used anymore and should be deleted